### ISSUE TITLE
6.4.4 (고정된 참조 위치 정보) 전자출판문서 형식을 어디까지 인정할 것인지에 대한 근거가 있을까요?

### ISSUE THREAD

**Issue by resistan** [a11ykr/kwcag22#1]
> 6.4.4 (고정된 참조 위치 정보) 전자출판문서 형식의 웹 페이지는 각 페이지로 이동할 수 있는 기능이 있어야 하고, 서식이나 플랫폼에 상관없이 참조 위치 정보를 일관되게 제공ㆍ유지해야 한다.
> 페이지 구분이 있는 전자출판문서 형식의 웹 페이지는 참조 위치 정보(예: 페이지 번호와 같은 페이 지 구분자(pagebreak locators))를 제공해야 하고, 각 페이지로 이동할 수 있는 기능도 제공해야 한다. 또한 콘텐츠의 확대/축소 등으로 서식이 변경되거나 플랫폼이 변경되어 참조 위치 정보가 사라지거 나 일관된 위치에 제공ㆍ유지되지 않을 경우, 참조 위치 정보를 사용하여 콘텐츠의 특정 부분을 지 칭해야 하는 상황(강의 등)에서 어려움이 있을 수 있기 때문에, 해당 참조 위치 정보는 서식이나 플 랫폼이 변경되더라도 일관된 위치에 제공ㆍ유지해야 한다.
> 
> 검사항목 6.4.4을 준수함으로써 얻을 수 있는 기대효과는 다음과 같다.
> 
> 콘텐츠를 확대해서 사용하는 사용자(저시력, 전맹, 인지장애 등)와 확대하지 않고 사용하는 사용자가 페이지 구분자(페이지 번호 등)를 사용하여 동일한 페이지를 참조할 수 있게 된다.

지침의 적용 범위나 검사 항목 내용으로는 PDF 등을 인정해야하는지 알 수가 없는데, 만약 EPub이나 PDF를 배제한다면 게시판 Paging 인터페이스 등 밖에 안남을 것 같습니다.

**Comment by a11ykr** [comment]
https://w3c.github.io/wcag/understanding/page-break-navigation

The scope of this criterion is web content that is part of a [Web page](https://w3c.github.io/wcag/understanding/page-break-navigation#dfn-web-page). EPUB can fulfill this definition if it is available to read at a URI. The more common case that is in scope is an EPUB book converted to be read by a web browser.

여기서 범위를 웹 페이지의 일부인 웹 콘텐츠로만 잡고 있고, EPUB는 URI에서 읽을 수 있는 경우만 적용합니다. 
웹 브라우저에서 읽을 수 있도록 변환된 EPUB 책입니다. -> 이건 아마 HTML로 변환된 경우를 말하는 것 같아요.

**Comment by mulder21c** [comment]
W3C github에서 이와 관련된 이슈를 찾아볼 수 있기는 한데요, 

[#2274](https://github.com/w3c/wcag/issues/2274)에서 질의된 내용과 유사한 논의 내용을 볼 수 있습니다.

[alastc의 답변](https://github.com/w3c/wcag/issues/2274#issuecomment-1080793600)을 보면    
> 다른 형식(예: ePub, PDF)에서는 사용자 에이전트가 탐색을 제공하지만 HTML에서는 제공하지 않습니다.

> ePub에는 적용되지 않지만 브라우저 기반 HTML용으로 용도가 변경된 ePub 문서는 범위에 포함됩니다.

라고 언급되는 부분이 있습니다.

여기에서 ePub이나 PDF 자체는 WCAG에서 관여하고자 하는 대상에서 제외되고, HTML 용으로 용도가 변경된 것만 범위에 포함시키려던 것으로 보여집니다.

다만, 게시판 Paging 인터페이스는 이 지침과 무관합니다.

[alastc의 답변](https://github.com/w3c/wcag/issues/2274#issuecomment-1080793600)에서 언급된 내용을 다시 살펴보면

> HTML 버전의 문서를 읽는 사람들에게 오프라인(또는 기타 디지털) 버전의 페이지 번호가 제공되는 경우에 사용됩니다.

라고 언급됩니다.    
게시판 paging 인터페이스는 여기에서부터 맞지 않습니다.      
이는 [#2274](https://github.com/w3c/wcag/issues/2274)에서 SC에 논의되던 Google Docs의 페이지 번호 기능과 유사합니다. (해당 이슈에서 논의된 바에서는 이러한 페이지 번호 기능은 이 지침과 관련 없음으로 종결됩니다)

**Comment by resistan** [comment]
물론 이게 권고안에 포함되는 건 아니지만, 실제로 많은 지침의 테크닉 쪽에는 PDF 관련 항목들이 있긴 하거든요. 일어나지 않은 일을 추측하는 게 좀 그렇긴 하지만 PDF는 포함하지 않을까요?

**Comment by mulder21c** [comment]
@resistan [#2026](https://github.com/w3c/wcag/issues/2026) 이 참고될 만한 이슈이지 않을가 싶은데요

여기서도 언급되는게 결국 ePub을 HTML로 변환할 때의 SC라서...
PDF가 HTML로 변환된 컨텐츠에 국한되지 않을까 싶습니다.

이슈 내용들에서 자주 언급되는 내용 중 하나가 PDF는 유저 에이전트에서 이미 제공하고 있다라는...

**Comment by miheeyaa** [comment]
> 여기서도 언급되는게 결국 ePub을 HTML로 변환할 때의 SC라서... PDF가 HTML로 변환된 컨텐츠에 국한되지 않을까 싶습니다.
> 
> 이슈 내용들에서 자주 언급되는 내용 중 하나가 PDF는 유저 에이전트에서 이미 제공하고 있다라는...


https://j-kagedu.or.kr/journal/view.php?number=1320
위 페이지의 PubReader 기능이 해당 항목의 점검 대상이 된다 봅니다. 
PDF로는 23페이지 고정값을 가지지만, pubreader로는 상대 값만 가지고 있고, 글자 확대/축소시에는 변경되고 있어 같은 페이지를 인식할 수 없고, 웹 페이지로 제공되고 있어서요.

**Comment by mulder21c** [comment]
@miheeyaa 네,해당 페이지에서 pubreader로 열어보면 PDF가 HTML로 변환 된 페이지가 제공되고 있으니, 이 경우에는 이 지침의 대상이 될 수 있을 것 같다는데 동의 합니다~
